ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -475,6 +475,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-ipsec
+- as: aws-ipi-ovn-ingress-nlb-p1-f4
```

Comment:
`aws-ipi-ovn-ingress-nlb` has been replace by `aws-ipi-ingress-lbtype-nlb` (as we also add `aws-ipi-ingress-lbtype-classic`), and we already have that in the file, so no need to add this one.

Reply:
ah, got it. I saw `aws-ipi-ovn-ingress-nlb` is also in arm nightly coverage, shall we replace it in arm ?

Reply:
Let arm take it, or use follow up PR to do that, so that we can get this one merge before the day off.

Reply:
fixed.